Define all the Connection ID fields (#202)
quicwg/load-balancers: draft-ietf-quic-load-balancers.md

packets. Server-generated connection IDs create a potential need for out-of-band
communication to support QUIC.

> QUIC allows servers (or load balancers) to designate an initial connection ID to

Reviewer: ```suggestion
QUIC allows servers (or load balancers) to select connection IDs so that the identifier
```

> encode useful routing information for load balancers.  It also encourages

Reviewer: ```suggestion
encodes useful routing information for load balancers.  It also encourages
```

Author: Um, I think the grammar here is wrong? The infinitive is "to encode"

> encode useful routing information for load balancers.  It also encourages

Author: ```suggestion
encodes useful routing information for load balancers.  It also encourages
```

# First CID octet {#first-octet}

> The first octet of a Connection ID is reserved for two special purposes, one

Reviewer: ```suggestion
The encoding scheme used in this document reserves the first octet of a Connection ID for two purposes: one
```

Local hardware cryptographic offload devices may accelerate QUIC servers by
receiving keys from the QUIC implementation indexed to the connection ID.

> However, on physical devices operating multiple QUIC servers, it is impractical

Author: ```suggestion
However, on physical devices operating multiple QUIC servers, it might be impractical
```

# Per-connection state

QUIC-LB requires no per-connection state at the load balancer. The load balancer
can extract the server ID from the connection ID of each incoming packet and
route that packet accordingly.

> However, once the routing decision has been made, the load balancer MAY

Reviewer: Not sure that this change is a positive one.

Author: OK

If the number of necessary server IDs per next hop is uniform, a simple
implementation would use successively longer server IDs at each tier of load

> balancing, and the server configuration would match the last tier. The forward

Reviewer: "forward" here meaning what?  Closer to the client?
